docs/book/src/roadmap.md

```diff
@@ -1,36 +1,21 @@
 # Cluster API Azure Roadmap
 
-This roadmap is a constant work in progress, subject to frequent revision. Dates are approximations. Features are listed in no particular order.
+The best place to see what's coming is in [the public milestones](https://github.com/kubernetes-sigs/cluster-api-provider-azure/milestones).  
+The next numbered milestone (e.g. **1.8**) is planned at the very beginning of the 2 month release cycle and most actively planned and discussed on the community call after the major release.
+Active community PR contributions are prioritized throughout the release and unplanned work may arise, hence the items in the milestone are a rough estimate which may change.
+The "next" milestone is a very rough collection of issues for the milestone after the current numbered one to help prioritize upcoming work.
 
-## v0.5 (v1alpha4) ~ Q1 2021
+## High Level Vision
 
-|Area|Description|Issue/Proposal|
-|--|--|--|
-|OS|Windows worker nodes|[#153](https://github.com/kubernetes-sigs/cluster-api-provider-azure/issues/153)|
-|Identity|Multi-tenancy within one manager instance|[#586](https://github.com/kubernetes-sigs/cluster-api-provider-azure/issues/586)|
-|UX|Bootstrap failure detection|[#603](https://github.com/kubernetes-sigs/cluster-api-provider-azure/issues/603)|
-|UX|Add tracing and metrics|[#311](https://github.com/kubernetes-sigs/cluster-api-provider-azure/issues/311)|
+CAPZ is the official production-ready Cluster API implementation to administer the entire lifecycle of self-managed or managed Kubernetes clusters (AKS) on Azure.   Cluster API provides a standard API and tooling consistent across on-premises and cloud providers to build and maintain Kubernetes clusters at large scale while working with GitOps and the surrounding tooling ecosystem.
```

Comment:
```suggestion
CAPZ is the official production-ready Cluster API implementation to administer the entire lifecycle of self-managed or managed Kubernetes clusters (AKS) on Azure. Cluster API extends the Kubernetes API to provide tooling consistent across on-premises and cloud providers to build and maintain Kubernetes clusters at scale while working with GitOps and the surrounding tooling ecosystem.
```

Reply:
Cecile wins this battle of English language density (but clear!)...